include/pros/misc.h

```diff
@@ -130,6 +131,17 @@ uint8_t competition_get_status(void);
  */
 #define competition_is_autonomous() ((competition_get_status() & COMPETITION_AUTONOMOUS) != 0)
 
+/**
+ * \return True if the V5 Brain is connected to VEXnet Field Controller, false otherwise.
+*/
+#define competition_is_field_control() ((competition_get_status() & COMPETITION_SYSTEM) != 0)
+
+/**
+ * \return True if the V5 Brain is connected to VEXnet Competition Switch, false otherwise.
+*/
+#define competition_is_competition_switch() ((competition_get_status() & COMPETITION_SYSTEM) == 0)
```

Comment:
we should come up with a better naming scheme for these, I know it's a legacy thing but man competition_is_competition is kind of repetitive.